e2e: EL7: TestE2E/PAR/INSPECT/inspect_command failure - unsquashfs not on PATH

**Version of Singularity**

main

**Describe the bug**

The `inspect_command` e2e-test failed, with `unsquashfs` not found on `$PATH`.

Not sure about this one... potentially a race? Are we modifying the environment anywhere else.

This was the sole failure on an EL7 e2e-test run

```
    --- FAIL: TestE2E (421.85s)
        --- FAIL: TestE2E/PAR (0.00s)
            --- FAIL: TestE2E/PAR/INSPECT (0.00s)
                --- FAIL: TestE2E/PAR/INSPECT/inspect_command (5.35s)
```

```
    === NAME  TestE2E/PAR/INSPECT/inspect_command
        exec.go:28: Running cmd "unsquashfs" with args ["-user-xattrs" "-d" "/tmp/stest.2567014158/inspect-1927781166/sandbox" "/tmp/stest.2567014158/inspect-1927781166/image.sqs"]
        inspect.go:78: Unexpected error while running command.
  
            Command:  unsquashfs -user-xattrs -d /tmp/stest.2567014158/inspect-1927781166/sandbox /tmp/stest.2567014158/inspect-1927781166/image.sqs
            ExitCode: 127
            Error:    exec: "unsquashfs": executable file not found in $PATH
            Stdout:
            Stderr:
```


**To Reproduce**
Steps to reproduce the behavior:

**Expected behavior**
A clear and concise description of what you expected to happen.

**OS / Linux Distribution**
Which Linux distribution are you using?
```
$ cat /etc/os-release
```

**Installation Method**
Write here how you installed SingularityCE. Eg. RPM, source.

**Additional context**
Anything else which might be relevant. E.g. if the bug only occurs on a specific filesystem, or kernel version etc.

--- a/e2e/inspect/inspect.go
+++ b/e2e/inspect/inspect.go
@@ -18,6 +18,7 @@ import (
 	"github.com/sylabs/singularity/v4/e2e/internal/testhelper"
 	"github.com/sylabs/singularity/v4/internal/pkg/test/tool/exec"
 	"github.com/sylabs/singularity/v4/internal/pkg/test/tool/require"
+	"github.com/sylabs/singularity/v4/internal/pkg/util/bin"
 	"github.com/sylabs/singularity/v4/pkg/image"
 	"github.com/sylabs/singularity/v4/pkg/inspect"
 )
@@ -73,7 +74,12 @@ func (c ctx) singularityInspect(t *testing.T) {
 
 	// First try with -user-xattrs since unsquashfs 4.4 gives an error code if
 	// it can't set system xattrs while rootless.
-	cmd := exec.Command("unsquashfs", "-user-xattrs", "-d", sandboxImage, squashImage)
+	unsquashfsPath, err := bin.FindBin("unsquashfs")
+	if err != nil {
+		t.Fatal("Unable to find 'unsquashfs' binary even though require.Command() was called")
+	}
+
+	cmd := exec.Command(unsquashfsPath, "-user-xattrs", "-d", sandboxImage, squashImage)
 	if res := cmd.Run(t); res.Error != nil {
 		t.Fatalf("Unexpected error while running command.\n%s", res)
 	}

